protocol/discord/types.go

```diff
@@ -1,17 +1,24 @@
 package discord
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/status-im/status-go/protocol/protobuf"
 )
 
-type ErrorType uint
+type ErrorCodeType uint
```

Comment:
Renamed this so we can have `WarningType` and `ErrorType` as enum below.
Reason I want those for the enum below is so we can have `Error()` and Warning` as function types.